homeassistant/components/broadlink/config_flow.py

```diff
@@ -41,7 +41,7 @@ async def async_set_device(self, device, raise_on_progress=True):
         """Define a device for the config flow."""
         supported_types = {
             device_type
-            for _, device_types in DOMAINS_AND_TYPES
+            for device_types in DOMAINS_AND_TYPES[1]
```

Comment:
This doesn't work. `DOMAINS_AND_TYPES` is a list of 2-item lists.

Reply:
should be `item[1] for item in DOMAINS_AND_TYPES`

Reply:
Should be solved in https://github.com/home-assistant/core/pull/42522/commits/ce4835bdd5a7b1caa5716defa021dd7f25ddcce8